src/main/java/redis/clients/jedis/JedisClusterCommand.java

```diff
@@ -85,7 +100,10 @@ public T runWithAnyNode() {
   }
 
   private T runWithRetries(final int slot) {
+    Instant deadline = Instant.now().plus(maxTotalRetriesDuration);
```

Comment:
It feels like we adding performance impact with calling now() on each command run.